queries/inko/highlights.scm

```diff
@@ -0,0 +1,169 @@
+; Brackets and operators
+[
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  "+"
+  "-"
+  "/"
+  "*"
+  "**"
+  "%"
+  "<"
+  ">"
+  "<="
+  ">="
+  "<<"
+  ">>"
+  ">>>"
+  "&"
+  "|"
+  "^"
+  "=="
+  "!="
+] @operator
+
+; Keywords
+[
+  "as"
+  "case"
+  "for"
```

Comment:
"as" -> `@keyword.import`
"case" -> `@keyword.conditional`
"for" -> `@keyword.repeat`
(maybe)

Reply:
@ObserverOfTime `as` is used for both type casts (`expr as SomeType`) and aliasing symbols in imports (`import some.module (SomeType as Whatever)`), hence I left it as a generic keyword.

`case` is used for pattern matching branches, so you're right in that that should be `keyword.conditional`, seems I forget that.

`for` is used for implementing traits, i.e. `impl SomeTrait for SomeClass { ... }`. There are no traditional for loops in Inko.